include/GafferSceneUI/TransformTool.h

```diff
@@ -255,6 +259,7 @@ class GAFFERSCENEUI_API TransformTool : public GafferSceneUI::SelectionTool
 		bool m_handlesDirty;
 
 		mutable std::vector<Selection> m_selection;
+		mutable bool m_selectionEditable;
```

Comment:
Knowing you're not normally a fan of storing duplicate state, is iterating `selection()` and determining this on-the-fly prohibitively expensive?

Reply:
I don't have any evidence for that really, so I'm not sure why I had it as a separate flag. 752184d .